sharedbuffers/mapped_struct.py

```diff
@@ -1831,19 +1832,24 @@ def _search_key(self, elem, dcode, offset, start, xlen, hint, equal=False):
         else:
             raise NotImplementedError("Unsupported data type for fast lookup: %s" % chr(dcode))
 
-    @cython.locals(lo=cython.Py_ssize_t, hi=cython.Py_ssize_t, step=cython.Py_ssize_t,
+    @cython.locals(eint=cython.ulonglong)
+    def _contains_compressed(self, elem):
```

Comment:
You want this to be a cfunc so it's a fast call, and you want in that case `elem` to be a native int already too so when you're looping over native ints you don't have to build a python int just to call this method